### PR TITLE
Refactor EvaluationMultiDSL#answer.

### PR DESCRIPTION
**Pull request by asherkach**

### CODE REVIEW

**Comment by asherkach** [comment]
This includes 6334c63 so as to avoid merge conflicts. Only the last commit should be considered part of this PR.

UPDATE: This PR has been rebased to only include the mentioned commit.

**Comment by asherkach** [comment]
CC @islemaster for CircleCI run and review.

File: dashboard/app/dsl/evaluation_multi_dsl.rb [hunk 1]
```diff
@@ -1,9 +1,11 @@
 class EvaluationMultiDSL < MultiDSL
   def answer(text, weight: 1, stage_name: nil)
-    answer = {text: text}
     raise 'Weight must be an integer' unless weight.is_a?(Integer)
-    answer[:weight] = weight || 1
-    answer[:stage] = stage_name
+    answer = {
+      text: text,
+      weight: weight || 1,
```
Comment (islemaster) [inline dashboard/app/dsl/evaluation_multi_dsl.rb:6]: Is `weight || 1` redundant with the default parameter value above?

Comment (asherkach) [reply dashboard/app/dsl/evaluation_multi_dsl.rb:6]: It is not redundant, specifically when `answer` is called with an explicit `weight` of `0`.

**Pull request merged**